components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -13617,10 +13617,11 @@ public Environment addEnvironment(String tenantDomain, Environment environment)
                     new String[]{DBUtils.getConvertedAutoGeneratedColumnName(dbProductName, "ID")})) {
                 prepStmt.setString(1, uuid);
                 prepStmt.setString(2, environment.getName());
-                prepStmt.setString(3, tenantDomain);
+                prepStmt.setString(3, "carbon.super");
```

Comment:
Better to use a constant here.

Reply:
Fixed with - https://github.com/wso2/carbon-apimgt/pull/11051